### PR TITLE
release 0.8.1 (#589)

### LINKED ISSUE #589: release 0.8.1

### PR DESCRIPTION
**Pull request by adietish**
fixes #589

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml
Comment (sbouchet) [inline src/main/resources/META-INF/plugin.xml:13]:
```suggestion
        <h3>0.8.1</h3>
```

**Review by sbouchet — CHANGES_REQUESTED** [review]

**Review by sbouchet — APPROVED** [review]
/lgtm

**Comment by adietish** [comment]
published to marketplace, waiting for approval
image

**Pull request merged**
